lib/feeder_importer.rb

```diff
@@ -71,25 +160,138 @@ def create_series(pcast = podcast)
     end
 
     self.distribution = Distributions::PodcastDistribution.create!(
-      url: feeder_podcast_url,
+      url: podcast_id,
```

Comment:
why just the id instead of `#{feeder_root}/podcasts/#{podcast_id}`?

Reply:
yeah, that looks wrong